HelloKelly/HelloKelly/Program.cs

```diff
@@ -8,15 +8,21 @@ namespace HelloKelly
 {
     class Program
     {
+        
         static void Main(string[] args)
         {
             string userName;
             Console.WriteLine("What is your name?");
             userName=Console.ReadLine();
-            Console.WriteLine("Nice to meet you, " + userName + "!");
+            Greeting(userName);
             Console.Write("Press <Enter> to exit... ");
             while (Console.ReadKey().Key != ConsoleKey.Enter) { }
 
         }
+        static void Greeting(string userName )
```

Comment:
To answer your question from your e-mail, yes - it's totally fine that the variable name is the same. From a technical perspective, each method has it's own "scope", which means the code inside a method, including the parameter names, are independent of the rest of the code in the file (unless you have a global variable/constant declared). 

From a design perspective, the name of the parameters should depend on how the method will be used. For example, a general Greeting method could be used to greet someone by username, by first name, by nickname, so if that were the case with our code then you could be more generic and call the parameter "string name". Then, each different method that calls the Greeting method will be passing in a string, locally named for it's functionality. Variable naming is something that is tough for everyone that you'll get better with as you grow :).